packages/soba/controls/src/lib/orbit-controls/orbit-controls.directive.ts

```diff
@@ -1,24 +1,18 @@
 import {
   EnhancedComponentStore,
+  makeVector3,
   NgtAnimationFrameStore,
   NgtEventsStore,
   NgtLoopService,
   NgtPerformanceStore,
   NgtStore,
   NgtVector3,
-  tapEffect,
+  tapEffect
 } from '@angular-three/core';
-import {
-  Directive,
-  EventEmitter,
-  Input,
-  NgModule,
-  NgZone,
-  OnInit,
-  Output,
-} from '@angular/core';
+import { Directive, EventEmitter, Input, NgModule, NgZone, OnInit, Output } from '@angular/core';
 import { Observable, Subscription, tap, withLatestFrom } from 'rxjs';
 import * as THREE from 'three';
+import { Vector3 } from 'three';
```

Comment:
nitpick: can we use `THREE.Vector3`?

Reply:
As I removed controlsTarget Type, we won't have this import anymore